### PR TITLE
Implemented splash screen and high fidelity login screen

### PR DESCRIPTION
**Pull request by tiffany-pann**
## Overview
- Implemented the splash screen while the app loads
- Updated the login screen to match high fidelity

### CODE REVIEW

File: Scoop/Supporting/Assets.xcassets/launchScreen.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```
Comment (vinnie4k) [inline Scoop/Supporting/Assets.xcassets/launchScreen.imageset/Contents.json:21]: Are these images actually scaled differently? There should be three different files all with different resolutions.

File: Scoop/Supporting/Assets.xcassets/loginBackground.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```
Comment (vinnie4k) [inline Scoop/Supporting/Assets.xcassets/loginBackground.imageset/Contents.json:21]: Same thing here.

File: Scoop/Supporting/Base.lproj/LaunchScreen.storyboard [hunk 3]
```diff
@@ -11,15 +14,27 @@
...
+            <point key="canvasLocation" x="52.307692307692307" y="374.64454976303318"/>
         </scene>
     </scenes>
+    <resources>
+        <image name="launchScreen" width="375" height="812"/>
+        <systemColor name="systemBackgroundColor">
+            <color white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+        </systemColor>
+    </resources>
 </document>
```
Comment (vinnie4k) [inline Scoop/Supporting/Base.lproj/LaunchScreen.storyboard:40]: Do you have any constraints for the launch screen images? Check on a different screen size.

**Review by vinnie4k — APPROVED** [review]
Looks good!

**Review by rs929 — APPROVED** [review]
Great Work Tiff! Looks good to me 🤩

File: Scoop/Supporting/Base.lproj/LaunchScreen.storyboard (hunk 3 above)
Comment (tiffany-pann) [reply Scoop/Supporting/Base.lproj/LaunchScreen.storyboard:40]: image
image
image

I adjusted some storyboard settings, seems okay? Do you know if this is how you test storyboard sizing

Comment (vinnie4k) [reply Scoop/Supporting/Base.lproj/LaunchScreen.storyboard:40]: ye looks good.

Comment (tiffany-pann) [reply Scoop/Supporting/Base.lproj/LaunchScreen.storyboard:40]: ok ty vin dow 🤲

**Pull request merged**